ignite/engine/__init__.py

```diff
@@ -560,6 +583,7 @@ def supervised_evaluation_step(
     device: Optional[Union[str, torch.device]] = None,
     non_blocking: bool = False,
     prepare_batch: Callable = _prepare_batch,
+    model_transform: Callable[[Any], Any] = lambda output: output,
```

Comment:
Actually, as you pointed out in another comment and I just remarked. I think `model_transform` is redundant here as `output_transform` already provide a similar functionality.